### PR TITLE
tests/conn_can: Add to README

### PR DESCRIPTION
**Pull request by JannesVolkens**
### Contribution description

This PR adds information back to the `README` that was previously removed.

### Testing procedure

None.

### Issues/PRs references

#12769
#13624

### CODE REVIEW

File: tests/conn_can/README.md [hunk 1]
```diff
@@ -3,6 +3,59 @@ tests/conn_can
...
+sudo apt-get install libsocketcan-dev:i386
+```
+On 32 bit you can just do the following:
+```
+sudo apt-get install libsocketcan-dev
+```
+
+Alternatively, you can compile from source:
+```
+wget http://www.pengutronix.de/software/libsocketcan/download/libsocketcan-0.0.10.tar.bz2
```
Comment (benpicco) [inline tests/conn_can/README.md:25]: Why not 
```suggestion
wget http://www.pengutronix.de/software/libsocketcan/download/libsocketcan-0.0.11.tar.bz2
```

File: tests/conn_can/README.md [hunk 2]
```diff
@@ -3,6 +3,59 @@ tests/conn_can
...
+
+$ sudo cd libsocketcan-0.0.10
+
+$ sudo ./configure
+
+compile in 32bits
+
+./configure --build=i686-pc-linux-gnu "CFLAGS=-m32" "CXXFLAG
+
+$ sudo make
```
Comment (benpicco) [inline tests/conn_can/README.md:39]:
```suggestion
$ tar xvjf libsocketcan-0.0.10.tar.bz2

$ rm libsocketcan-0.0.10.tar.bz2

$ cd libsocketcan-0.0.10

$ ./configure

compile in 32bits

./configure --build=i686-pc-linux-gnu "CFLAGS=-m32" "CXXFLAG

$ make
```

why `sudo`?

File: tests/conn_can/README.md [hunk 3]
```diff
@@ -22,21 +22,21 @@ sudo apt-get install libsocketcan-dev
 
 Alternatively, you can compile from source:
 ```
-wget http://www.pengutronix.de/software/libsocketcan/download/libsocketcan-0.0.10.tar.bz2
+wget http://www.pengutronix.de/software/libsocketcan/download/libsocketcan-0.0.11.tar.bz2
 
-$ sudo tar xvjf libsocketcan-0.0.10.tar.bz2
+$ tar xvjf libsocketcan-0.0.10.tar.bz2
```
Comment (benpicco) [inline tests/conn_can/README.md:27]: Well now you also need to change the version number here ;)

**Comment by JannesVolkens** [comment]
@benpicco Thank you for your review, I addressed your comments.

File: tests/conn_can/README.md (hunk 3 above)
Comment (JannesVolkens) [reply tests/conn_can/README.md]: You're right! Thank you for the hint.

**Review by PeterKietzmann — APPROVED** [review]
This PR adds (and updates!) documentation that has accidentally been removed in #12769. ACK

**Comment by PeterKietzmann** [comment]
Please squash!

**Pull request merged**